src/styles/styles.js

```diff
@@ -2761,6 +2761,20 @@ const styles = {
         fontSize: variables.fontSizeSmall,
         ...spacing.ph2,
     },
+
+    workspaceSection: {
```

Comment:
I think this is fine for now, but I could see making this even more generic... like `cardSection` or something? In case we use it elsewhere

Reply:
yeah I like that too. This was already more genericized than my first attempt (`workspaceEditorSection`) but will make it even broader.